### PR TITLE
Saves & Follows view

### PR DESCRIPTION
**Pull request by l2succes**
Fixes https://github.com/artsy/collector-experience/issues/700

This PR includes the new Saves & Follows view including adding a new external VC to emission to be imported into eigen. Still need to add tests and types.

Tabs completed
- [x] Works
- [x] Artists
- [x] Categories

Skip New Tests

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/__tests__/TabBar-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/Components/__tests__/SavedArtistRow-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/Relay/__tests__/ArtistsRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Works/Relay/__tests__/WorksRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Works/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/__tests__/TabBar-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/Components/__tests__/SavedArtistRow-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/Relay/__tests__/ArtistsRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Works/Relay/__tests__/WorksRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Works/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by orta** [comment]
If you can get this green, and you're happy, we can get this in partially 👍

**Comment by orta** [comment]
https://github.com/artsy/metaphysics/pull/813 adds Genes to metaphysics

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/__tests__/TabBar-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/Components/__tests__/SavedArtistRow-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/Relay/__tests__/ArtistsRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artworks/Relay/__tests__/ArtworksRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artworks/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Scenes/Favorites/Components/Categories/index.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import { View } from "react-native"
+
+class Categories extends React.Component<null, null> {
```
Comment (l2succes) [inline src/lib/Scenes/Favorites/Components/Categories/index.tsx]: @orta This is the placeholder component where the Categories tab view should go

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/__tests__/TabBar-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/Relay/__tests__/ArtistsRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artworks/Relay/__tests__/ArtworksRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artworks/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Categories/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Works/Relay/__tests__/WorksRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Works/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/__tests__/TabBar-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/Relay/__tests__/ArtistsRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artworks/Relay/__tests__/ArtworksRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artworks/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Categories/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Works/Relay/__tests__/WorksRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Works/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Scenes/Favorites/Components/Categories/index.tsx (hunk 1 above)
Comment (orta) [reply src/lib/Scenes/Favorites/Components/Categories/index.tsx]: Rock

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/__tests__/TabBar-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/Relay/__tests__/ArtistsRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artworks/Relay/__tests__/ArtworksRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artworks/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Categories/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Works/Relay/__tests__/WorksRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Works/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/__tests__/TabBar-tests.tsx`
- `src/lib/Components/Lists/__tests__/SavedItemRow-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/Relay/__tests__/ArtistsRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artworks/Relay/__tests__/ArtworksRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artworks/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Categories/Relay/__tests__/CategoriesRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Categories/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by orta** [comment]
OK, this is at a good place. I've made https://github.com/artsy/collector-experience/issues/718 to handle going from the current query renderers to pagination renderers, shouldn't be too much effort but I'd like to get this in.

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/__tests__/TabBar-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/Relay/__tests__/ArtistsRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artworks/Relay/__tests__/ArtworksRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artworks/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Categories/Relay/__tests__/CategoriesRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Categories/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/__tests__/TabBar-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/Relay/__tests__/ArtistsRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artists/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artworks/Relay/__tests__/ArtworksRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artworks/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Categories/Relay/__tests__/CategoriesRenderer-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Categories/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**